visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -94,11 +94,31 @@ export class CustomConfigHelper {
 	}
 
 	private static loadCustomConfigs() {
+		const ccLocalStorage = this.getCcLocalStorage()
+		return new Map(ccLocalStorage?.customConfigs)
+	}
+
+	// TODO [2022-08-01]: remove replace method for SINGLE mode before when deadline is reached
+	private static getCcLocalStorage() {
 		const ccLocalStorage: LocalStorageCustomConfigs = JSON.parse(
 			CustomConfigHelper.getStorage().getItem(CUSTOM_CONFIGS_LOCAL_STORAGE_ELEMENT),
 			stateObjectReviver
 		)
-		return new Map(ccLocalStorage?.customConfigs)
+
+		if (localStorage?.getItem("codeChartaVersion") < "1.88.0") {
```

Comment:
I think this will not work since the version is set to the current one as soon as CodeCharta is loaded, isn't it?
If so, I would increase the version of `CUSTOM_CONFIGS_LOCAL_STORAGE_VERSION ` to `1.0.1` after the local storage elements has been rewritten from Single to Multiple mode. Then, it would be easy to check old configs (`=== 1.0.0`).

Reply:
You're right. Sounds much better :)